openapi-core/src/main/java/io/ballerina/openapi/core/generators/service/ParametersGenerator.java

```diff
@@ -81,8 +85,14 @@ public class ParametersGenerator {
     private final List<Node> requiredParams;
     private final List<Node> defaultableParams;
 
-    public ParametersGenerator(boolean isNullableRequired) {
+    private final OpenAPI openAPI;
+
+    private static final List<String> queryParamSupportedTypes =
+            new ArrayList<>(Arrays.asList(GeneratorConstants.INTEGER, GeneratorConstants.NUMBER,
+                    GeneratorConstants.STRING, GeneratorConstants.BOOLEAN));
+    public ParametersGenerator(boolean isNullableRequired, OpenAPI openAPI) {
```

Comment:
Missing newline

Reply:
Resolved in https://github.com/ballerina-platform/openapi-tools/pull/1335/commits/445e23f5bd95629f18e244673178a695e7d65eda